src/main/java/org/folio/edge/orders/OrdersHandler.java

```diff
@@ -57,19 +59,12 @@ protected void handleCommon(RoutingContext ctx, String[] requiredParams, String[
   }
 
   protected void handle(RoutingContext ctx) {
-    handleCommon(ctx,
-        new String[] {},
-        new String[] {},
-        (client, params) -> {
-          PurchasingSystems ps = PurchasingSystems.fromValue(params.get(PARAM_TYPE));
-          logger.info("Request is from purchasing system: " + ps.toString());
-          ((OrdersOkapiClient) client).placeOrder(
-              ps,
-              ctx.getBodyAsString(),
-              ctx.request().headers(),
-              resp -> handleProxyResponse(ps, ctx, resp),
-              t -> handleProxyException(ctx, t));
-        });
+    handleCommon(ctx, new String[] {}, new String[] {}, (client, params) -> {
+      PurchasingSystems ps = PurchasingSystems.fromValue(params.get(PARAM_TYPE));
+      logger.info("Request is from purchasing system: " + ps.toString());
+      ((OrdersOkapiClient) client).placeOrder(ps, ctx.getBodyAsString(), ctx.request()
+          .headers(), resp -> handleProxyResponse(ps, ctx, resp), t -> handleProxyException(ctx, t));
+    });
```

Comment:
I agree it is difficult to visually read , but this formatting follows the java conventions(built-in Eclipse), and folio aligns with the java conventions for formatting. 
Me and Craig were discussing on this yesterday, like the stylesheet for order import, is there a story for standardizing formatting too? it would be nice to import a standard formatter across.

Reply:
My view is don't reformat code unless it is code you are adding and, even then, it is better to follow the convention of the existing file. I think, if we agree to reformat a file, it should be included in some sort of technical debt story so that actual logic changes are not missed due to reformatting.

If someone went out of their way to set up formatting that makes the code easier to grok, squashing it into a single line, because eclipse thinks it is a good idea, is probably not the best reasoning.

I don't think we'll ever get to a standard format, since there are a wide variety of editors and developer preferences. But if we adopt the approach of don't touch lines that have nothing to do with your story, then we can eliminate some of the PR review pain as well as format wars between IDEs.

Does that make sense?

Reply:
It makes sense, but I have some other thoughts on this. I'll take it up later with you.